src/main.opy

```diff
@@ -2,6 +2,61 @@
 
 #!include "settings.opy"
 
+
+playervar hero
```

Comment:
I believe there already is a variable that also sets the current hero `HeroActive`

Reply:
Probably ye, I saw something about this, however I saw that HeroActive can be set to "null".